src/routes/search/+page.svelte

```diff
@@ -0,0 +1,111 @@
+<script lang="ts">
+  import "./search-page.scss";
+  import { onMount } from "svelte";
+  import algoliasearch from "algoliasearch";
+  import instantsearch from "instantsearch.js";
+  import { configure, hits, searchBox, stats, pagination } from "instantsearch.js/es/widgets";
+  import { searchHitsTemplate } from "./searchHelpers";
+
+  const ALGOLIA_APP_ID = "9IIDYROXZ5";
+  const ALGOLIA_API_KEY = "0eac885faaa4c70dfc9dd8b6ab4ab10f";
+  onMount(() => {
+    const search = instantsearch({
+      searchClient: algoliasearch(ALGOLIA_APP_ID, ALGOLIA_API_KEY),
+      // TODO: change this the "contentful" index when ready (name subject to change)
+      indexName: "media-sample-data",
+      routing: true,
+      searchFunction(helper) {
+        const searchUI = document.querySelectorAll(".stats, .hits, .pagination");
+
+        searchUI.forEach((element) => {
+          (element as HTMLElement).style.display = helper.state.query === "" ? "none" : "";
+        });
+
+        helper.search();
+      },
+    });
+
+    search.addWidgets([
+      configure({
+        hitsPerPage: 3,
+        attributesToSnippet: ["content:14"],
```

Comment:
Is there a reason we're marking this lower than the default of `80`?

Reply:
I think I was just trying to slim down the results as I was working with it at first, but that might just be a product of the sample data. E.g., if we show `hit._highlightResult.content.value` instead of `hit._snippetResult.content.value`, it's a bit more verbose:
![2e81d6052505577e272520b354e15cca](https://github.com/la-ldaf/ldaf-site/assets/13280995/988d6653-4cf2-4a75-b702-b61127c8f229)

Reply:
^ I don't mind switching it back to that for the time being. I imagine the LDAF data will be more concise and this problem might not even exist for it

Reply:
This might go a bit too far in the other direction; I think we should try to limit the highlighted / snippeted content to just a few lines, but we can handle that once we're working with real data.